### DISCUSSION TITLE
Filter out guest users

### DISCUSSION THREAD

**Discussion by ShareThePoint** [microsoft-search/pnp-modern-search#2681] [Q&A] [answered]
I am using searchbox and searchresults to get a simple people search and I don't want to see guest accounts.

How do I create a filter?

![2023-01-05 16_51_09-Intranet – Startside](https://user-images.githubusercontent.com/46026545/210828090-1cc93b91-1940-45c4-bb23-3c20fddbf01b.jpg)

BR. Allan

**Comment by kasperbolarsen** [comment, marked as answer]
Hi Allan, loads of way to do that, like -Workemail:EXT , but perhaps you would prefer to select the profiles you want to see using something like Workemail:youdomain.dk as this also excludes accounts without an email.

**Reply by ShareThePoint** [reply]
Thank you very much @kasperbolarsen
